packages/excalidraw/clipboard.ts

```diff
@@ -435,7 +434,7 @@ export const copyTextToSystemClipboard = async (
 
   // (3) if that fails, use document.execCommand
   if (!copyTextViaExecCommand(text)) {
-    throw new Error(errorMessage || "Error copying to clipboard.");
+    throw new Error();
```

Comment:
let's keep the default `"Error copying to clipboard."` for host apps or us in the future if we forget to catch